src/server/main.py

```diff
@@ -1,26 +1,66 @@
 from flask import Blueprint, Flask
 from flask_cors import CORS
+from werkzeug.serving import BaseWSGIServer, make_server
 
 from .auth import AuthController
 from .database import DatabaseProvider
 from .stock_market import StockMarketController
 
-DatabaseProvider.initialize()
 
-app = Flask(__name__)
+class Server:
+    """Main application start point."""
 
-api = Blueprint("api", __name__, url_prefix="/api")
-v1 = Blueprint("v1", __name__, url_prefix="/v1")
+    def __init__(self) -> None:
+        """Initialize server application along with its endpoints and cors."""
+        DatabaseProvider.initialize()
 
-v1.register_blueprint(AuthController.blueprint)
-v1.register_blueprint(StockMarketController.blueprint)
-api.register_blueprint(v1)
-app.register_blueprint(api)
+        self.name: str = __name__
+        self.app: Flask = self._create_app()
+        self.server: BaseWSGIServer = make_server("127.0.0.1", 5000, self.app)
```

Comment:
What is this 5000 magic number? Please either add comment in code explaining this value or use a variable.

Reply:
I removed this WSGI server entirely because we use gunicorn anyway and there's no need to launch the server programmatically.